components/Tagline.js

```diff
@@ -17,26 +17,28 @@ const TaglineText = styled.p`
   margin-top: ${({ hideTopRuler }) => (hideTopRuler ? '0' : taglineTextMargin)};
 `
 
-const Tagline = ({ text, hideTopRuler, source }) => (
-  <PageSection>
-    {hideTopRuler ? null : <Divider color="orange" />}
-    <TaglineText hideTopRuler={hideTopRuler}>
-      {text}
-      {source}
-    </TaglineText>
-    <Divider color="orange" />
-  </PageSection>
-)
+const Tagline = ({ text, hideTopRuler, source }) =>
+  text && (
```

Comment:
Please make sure that `Tagline` returns `null` instead of undefined if `text` is missing. This is currently breaking the build:

https://reactjs.org/docs/error-decoder.html?invariant=152&args[]=Tagline